image.yaml

```diff
@@ -6,7 +6,7 @@ version: "1.4"
 from: "openshift3/apb-base:v3.11"
 labels:
   - name: "com.redhat.component"
-    value: "rhpam-apb"
+    value: "rhpam-7-rhpam76-apb-openshift-apb"
```

Comment:
Shouldn't that be `rhpam-7-rhpam76-apb-openshift-container` ?

Reply:
It is part of CPaaS project, David sent a email about it.

Reply:
ignore my comment :+1:

Reply:
@ruromero , please see my comment here:

https://projects.engineering.redhat.com/browse/RCM-65840?focusedCommentId=1847345&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1847345

I will also put it in the email thread. @spolti : FYI